app/models/cma_case.rb

```diff
@@ -0,0 +1,29 @@
+class CmaCase < Document
+  FORMAT_SPECIFIC_FIELDS = [
+    :opened_date,
+    :closed_date,
+    :case_type,
+    :case_state,
+    :market_sector,
+    :outcome_type
+  ]
+
+  attr_accessor *FORMAT_SPECIFIC_FIELDS
+
+  def initialize(params = {})
+    super(params, FORMAT_SPECIFIC_FIELDS)
+  end
+
+  def format
+    "cma_case"
+  end
+
+  def public_path
+    "/cma-cases"
+  end
+
+  def organisations
+    # Org name: Competition And Markets Authority
+    ["957eb4ec-089b-4f71-ba2a-dc69ac8919ea"]
```

Comment:
It would be good to talk to finding-things about this - we're planning on making organisation tagging possible centrally using the new content-tagger app (turning it on format-by-format as we make the publisher apps compatible), but setting a sensible default is important.  It seems most likely to me that copying the org links from the associated finder content item is the best approach for new specialist documents.